ArmoryQt.py

```
@@ -1815,13 +1823,14 @@ def loadSettings(self):
 
    #############################################################################
    def loadWallets(self):
-      def loadWltsLbd():
-         wltList = TheBridge.wltManager.listWallets()
-         wltsProto = TheBridge.wltManager.loadWallets()
-         self.wallets.setupFromProto(wltsProto)
+      try:
```

Why is this in a try block if you do not handle the error? Just let the exception propagate if you do not know what to do with it.